dynamic-property-polling/build.gradle.kts

```diff
@@ -13,7 +13,11 @@ dependencies {
     implementation(Libs.jfix_stdlib_concurrency){
         exclude("ru.fix")
     }
+    implementation(Libs.jfix_stdlib_reference){
+        exclude("ru.fix")
```

Comment:
is it required to declare exclude block explicitly? As I see, stdlib-reference doesn't have any jfix dependencies

https://github.com/ru-fix/jfix-stdlib/blob/release/2.0/jfix-stdlib-reference/build.gradle.kts